packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -108,7 +111,7 @@ def invoke_detect_ip():
 connect-port=2888
 observer-threshold=0
 election-port=3888
-zoo-cfg-extra=tickTime\\=2000&initLimit\\=10&syncLimit\\=5&quorumListenOnAllIPs\\=true&maxClientCnxns\\=0&autopurge.snapRetainCount\\=5&autopurge.purgeInterval\\=6
+zoo-cfg-extra=tickTime\\=2000&initLimit\\=10&syncLimit\\=5&quorumListenOnAllIPs\\=true&maxClientCnxns\\=0&autopurge.purgeInterval\\=1
```

Comment:
Do you think for clarity to keep the `autopurge.snapRetainCount\\=3` that is the default?

Reply:
It's a tricky one. On one hand, as you suggest, explicitly stating the defaults indicates that the value was chosen purposefully, even though it was the default. On the other hand, it could be clearer to keep the config short to specify "the defaults, with these few purposefully chosen exceptions".

In this case, if ZooKeeper found a reason to change the number of retained snapshots, we would probably go with their findings. Hence, we accept the default, whatever it may be, rather than specifically needing 3 snapshots.